src/converters/light.ts

```diff
@@ -390,6 +460,38 @@ class LightHandler implements ServiceHandler {
     }
     return identifier;
   }
+
+  private handleAdaptiveLighting(value: number): boolean {
+    // Adaptive Lighting active?
+    if (this.adaptiveLighting !== undefined && this.adaptiveLighting.isAdaptiveLightingActive()) {
+      if (this.lastAdaptiveLightingTemperature === undefined) {
+        this.lastAdaptiveLightingTemperature = value;
+      } else {
+        const change = Math.abs(this.lastAdaptiveLightingTemperature - value);
+        if (change < 1) {
+          this.accessory.log.debug(
+            `Adaptive Lighting: Color temperature ${value} skipped for ${this.accessory.displayName}. ` +
+              `Previous: ${this.lastAdaptiveLightingTemperature}`
+          );
+          return false;
+        }
+
+        this.accessory.log.info(`Adaptive Lightning: Update for ${this.accessory.displayName} temperature=${value}`);
```

Comment:
Perhaps move this to the `debug` level for now. Might remove it later on depending on how much noise it generates.

Reply:
Changed to debug level.